spec/classes/init_spec.rb

```diff
@@ -1661,6 +1663,25 @@
     end
   end
 
+  context 'with sshd_kerberos_authentication set to invalid value on valid osfamily' do
```

Comment:
so similar to this?
```puppet
describe 'with KerberosAuthentication' do
  ['yes','no','invalid'].each do |value|
    context "set to #{value}" do
      # set param to value
      it { should contain_file('sshd_config').with_content(/^KerberosAuthentication #{value}$/) }
    end
  end
end
```

I'm a little lost or don't bother with the invalid check?